lib/utils.js

```diff
@@ -41,9 +41,16 @@ module.exports = {
          args[0] = currentRepo;
          return singleRepoLambda.apply(this, args);
       };
-      return Promise.all(config.repos.map(allRepoMap))
+
+      return Promise.all(this.getRepoFullNames().map(allRepoMap))
       .then(function(repoItems) {
          return _.flatten(repoItems, /* shallow */ true);
       });
-   }
+   },
+
+   getRepoFullNames: function() {
+      return config.repos.map(function(repo) {
```

Comment:
Now that the dev environment has node v6 this can be

```js
return config.repos.maps(repo => repo.full_name);
```

:D

Reply:
Planning on doing this

Reply:
5deb403